example/tests/integration/test_includes.py

```diff
@@ -31,7 +31,7 @@ def test_included_data_on_detail(single_entry, client):
 
 def test_dynamic_related_data_is_included(single_entry, entry_factory, client):
     entry_factory()
-    response = client.get(reverse("entry-detail", kwargs={'pk': single_entry.pk}) + '?include=suggested')
+    response = client.get(reverse("entry-detail", kwargs={'pk': single_entry.pk}) + '?include=featured')
```

Comment:
@jerel why was the test changed here? I get a failure when including a SerializerMethodResourceRelatedField, is this intentional? 

The failure is `'function' object has no attribute 'parent'`